bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/SecurityApi.java

```diff
@@ -66,7 +69,7 @@ public boolean addwebhook(URI uri) throws NetatmoException {
         return true;
     }
 
-    private Collection<HomeEvent> getEvents(@Nullable Object... params) throws NetatmoException {
+    private List<HomeEvent> getEvents(@Nullable Object... params) throws NetatmoException {
         UriBuilder uriBuilder = getApiUriBuilder(SUB_PATH_GET_EVENTS, params);
         BodyResponse<Home> body = get(uriBuilder, NAEventsDataResponse.class).getBody();
```

Comment:
You were talking about 30 events returned. Is it a number fixed by the API ?

Reply:
Yes, by default they are paged by batches of 30 (this can be changed using the `size`parameter

Reply:
We have to avoid the recursion as much as possible. Do you think 30 is enough to cover 95% of the normal usages (including the case you own several cameras) or do you think we should increase it?

Reply:
It is hard to tell. The number of events presented between two polls depends highly upon setup size and the number of devices emitting events. We could implement a kind of recursion counter that would increase page size dynamically. Maybe in a second step ?

Reply:
Yes, can be improved later